resources/sagemaker_training_jobs.go

```diff
@@ -0,0 +1,888 @@
+				Columns: []schema.Column{
+					{
+						Name:        "training_job_cq_id",
+						Description: "Unique CloudQuery ID of aws_sagemaker_training_jobs table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "channel_name",
+						Description: "The name of the channel.  This member is required.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "data_source_file_system_data_source_directory_path",
+						Description: "The full path to the directory to associate with the channel.  This member is required.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DataSource.FileSystemDataSource.DirectoryPath"),
+					},
+					{
+						Name:        "data_source_file_system_data_source_file_system_access_mode",
+						Description: "The access mode of the mount of the directory associated with the channel",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DataSource.FileSystemDataSource.FileSystemAccessMode"),
+					},
+					{
+						Name:        "data_source_file_system_data_source_file_system_id",
+						Description: "The file system id.  This member is required.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DataSource.FileSystemDataSource.FileSystemId"),
+					},
+					{
+						Name:        "data_source_file_system_data_source_file_system_type",
+						Description: "The file system type.  This member is required.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DataSource.FileSystemDataSource.FileSystemType"),
+					},
+					{
+						Name:        "data_source_s3_data_source_s3_data_type",
+						Description: "If you choose S3Prefix, S3Uri identifies a key name prefix",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DataSource.S3DataSource.S3DataType"),
+					},
+					{
+						Name:        "data_source_s3_data_source_s3_uri",
+						Description: "Depending on the value specified for the S3DataType, identifies either a key name prefix or a manifest",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DataSource.S3DataSource.S3Uri"),
+					},
+					{
+						Name:        "data_source_s3_data_source_attribute_names",
+						Description: "A list of one or more attribute names to use that are found in a specified augmented manifest file.",
+						Type:        schema.TypeStringArray,
+						Resolver:    schema.PathResolver("DataSource.S3DataSource.AttributeNames"),
+					},
+					{
+						Name:        "data_source_s3_data_source_s3_data_distribution_type",
+						Description: "If you want Amazon SageMaker to replicate the entire dataset on each ML compute instance that is launched for model training, specify FullyReplicated",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DataSource.S3DataSource.S3DataDistributionType"),
+					},
```

Comment:
the name here seems a bit redundant, for example data_source_s3_data_source_s3_data...  -> s3_data_source_distribution_type